std/numeric.d

```diff
@@ -3088,3 +3088,56 @@ unittest
     static assert(isComplexLike!(Complex!double));
     static assert(!isComplexLike!(uint));
 }
+
+/**
+Computes the $(WEB en.wikipedia.org/wiki/Binomial_coefficient, binomial coefficient)
+of n and k.
+It is also known as "n choose k" or more formally as $(D _n!/_k!(_n-_k)).
+
+Params:
+    n = non negative integer
+    k = non negative integer, k <= n
+
+Returns:
+    binomial coefficient
+*/
+
+public:
+size_t binomial(size_t n, size_t k) pure nothrow @safe @nogc
+in
+{
+    assert(n > 0, "binomial: n must be > 0.");
```

Comment:
This `assert` is unnecessary because `n == 0` is also valid (as documented above).